sgkit_plink/pysnptools.py

```diff
@@ -122,13 +122,17 @@ def coerce_code(v, codes):
 def read_bim(path: PathType, sep: str = "\t") -> DataFrame:
     # See: https://www.cog-genomics.org/plink/1.9/formats#bim
     names = [f[0] for f in BIM_FIELDS]
-    df = dd.read_csv(str(path) + ".bim", sep=sep, names=names, dtype=BIM_DF_DTYPE)
+    df = dd.read_csv(str(path), sep=sep, names=names, dtype=BIM_DF_DTYPE)
     df["contig"] = df["contig"].where(df["contig"] != "0", None)
     return df
 
 
 def read_plink(
-    path: PathType,
+    *,
```

Comment:
This seems like a solid and unambiguous way to do it. One nice thing about supporting the single positional arg form was that would generalise pretty well across different formats - ``read_vcf``, ``read_bgen`` etc should all work with a single argument (most of the time). But, once we're consistent with the kwarg being ``path`` across the different formats then this will generalise just as well.